app/code/Magento/Theme/view/frontend/layout/print.xml

```diff
@@ -16,5 +16,6 @@
         <referenceContainer name="before.body.end">
             <block class="Magento\Framework\View\Element\Template" name="sales.order.print.js" template="Magento_Theme::html/print.phtml" />
         </referenceContainer>
+        <referenceBlock name="authentication-popup" remove="true"/>
```

Comment:
I see this block is defined in the Magento/Customer module. Could you move your change to this module as well?
https://github.com/magento/magento2/blob/a360c06684322189609dd543d09bc31d8d2bdca0/app/code/Magento/Customer/view/frontend/layout/default.xml#L26

Reply:
Sure @ihor-sviziev

Reply:
@ihor-sviziev Your suggested changes were done. Will you please review & proceed with my PR?

Reply:
Hi @pratiksangani,
Now we basically removing this block in two modules that is not correct. We should remove it only in customer module, there should not be any changes in “theme” module